en/components/banner.md

```diff
@@ -83,7 +83,7 @@ Changing the message displayed in the banner is easy - just change the content y
 
 #### Adding an icon
 
-Developers can pass an [`igx-icon`]({environment:angularApiUrl}/classes/igxiconmodule.html) in the banner's content and it will always be positioned at **the beginning** of the banner message.
```

Comment:
This could be a link to the icon topic actually.

Reply:
Added a link to the icon topic on the first mention